app/src/views/private/components/module-bar-logo.vue

```diff
@@ -12,7 +12,7 @@
 			<transition name="fade">
 				<v-progress-linear v-if="showLoader" indeterminate rounded @animationiteration="stopSpinnerIfQueueIsEmpty" />
 			</transition>
-			<v-image class="custom-logo" :src="customLogoPath" alt="Project Logo" />
+			<img class="custom-logo" :src="customLogoPath" alt="Project Logo" />
```

Comment:
I think you forgot to add the `addTokenToURL` function to the `customLogoPath` computed.

Reply:
The `project_logo` is publicly accessible, thus there's no need for the token to be added.

https://github.com/directus/directus/blob/89142d82bce614fc82a0168336c502aae8e365b9/api/src/services/assets.ts#L46-L51

https://github.com/directus/directus/blob/89142d82bce614fc82a0168336c502aae8e365b9/api/src/services/assets.ts#L62-L64